_query-dsl/specialized/index.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: default
+title: Specialized queries
+has_children: true
+nav_order: 65
+has_toc: false
+---
+
+# Specialized queries
+
+OpenSearch supports the following specialized queries:
+
+- `distance_feature`: Calculates document scores based on the dynamically calculated distance between the origin and a document's `date`, `date_nanos`, or `geo_point` fields. This query can skip non-competitive hits.
+
+- `more_like_this`: Finds documents similar to the provided text, document, or collection of documents.
+
+- `percolate`: Finds queries (stored as documents) that match the provided document.
+
+- `rank_feature`: Calculates scores based on the values of numeric features. This query can skip non-competitive hits.
+
+- `script`: Uses a script as a filter.
+
+- `script_score`: Lets you modify a subquery score by using a custom script.
```

Comment:
I would not use `subquery` here which is a little confusing. 

Maybe revise from "Uses a script to provide a custom score for returned documents."